src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -130,26 +130,20 @@ def _handle_global_configuration():
         global_config.set('logging', 'enable_log_file', 'yes' if enable_file_logging else 'no')
         set_global_config(global_config)
 
-def handle_configure(section=None, name=None, value=None, default_resource_group=None):
+def handle_configure(setting=None, value=None):
     '''
     configure common settings
-    :param str section: configuration section
-    :param str name:    configuration variable name
+    :param str setting: configuration variable, e.g. resource-group, appservice/webapp, compute/vm
```

Comment:
I initially used `resource-group`, but switched to `resource_group`, as the existing variables there all use `_` style. I would leave to @derekbekoe for final comment.
Regarding to `group`, I am fine with that with the only concern that by appearing in the global config, the meaning is not easy to grab, as the `group` could mean lots of other things